src/main/java/com/google/opensesame/servlets/MentorObject.java

```diff
@@ -1,19 +1,43 @@
 package com.google.opensesame.servlets;
 
-public class MentorObject {
-  private String name;
-  private String description;
+import java.util.ArrayList;
 
-  public MentorObject(String nameInput, String descriptionInput) {
-    name = nameInput;
-    description = descriptionInput;
+public class MentorObject extends PersonObject {
+  private ArrayList<String> projectIDs;
+
+  public MentorObject(String nameInput, String gitHubIDInput) {
```

Comment:
In general avoid using `Input` in variable names. Usually the word `input` doesn't tell me any additional information about what the variable will be used for or the information it is holding.

```suggestion
  public MentorObject(String name, String gitHubID) {
```

Reply:
I'm only commenting on this method because I don't want to repeat myself, but this advice applies to all methods in this file, not just this one.

Reply:
Thank you!